docker_custodian/docker_gc.py

```diff
@@ -21,6 +22,17 @@
 YEAR_ZERO = "0001-01-01T00:00:00Z"
 
 
+def list_events_since(client, since):
+    """
+    Retrieves a list of events that have happened "since"
+    the datetime provided or that happend in the last x seconds.
+    :param client: docker client
+    :param since: timedelta
+    """
+    events = client.events(since=dt.utcnow() - since, until=1, decode=True)
+    return [event for event in events]
```

Comment:
Events `since` is not really reliable.

https://github.com/docker/docker/blob/master/daemon/events/events.go#L12 

It only stores the last 64 events, so on a heavily used server, it's probably not going to have the events you want.